src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+import model.Colors;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+
+        return color.toString();
```

Comment:
Instead of creating a new instance of `Random` each time the method is called, it would be more efficient to create a single static instance of `Random` and reuse it. This is a performance issue because creating a new `Random` object each time is unnecessary and can be costly if the method is called frequently.